src/main/kotlin/uk/gov/dluhc/printapi/service/aed/AnonymousElectorDocumentService.kt

```diff
@@ -86,6 +92,14 @@ class AnonymousElectorDocumentService(
             .map { anonymousElectorDocumentMapper.mapToAnonymousElectorDocumentDto(it) }
     }
 
+    @Transactional(readOnly = true)
+    fun getAnonymousElectorDocumentsByApplicationId(applicationId: String): List<AnonymousElectorDocumentDto> {
+        return anonymousElectorDocumentRepository.findBySourceTypeAndSourceReference(
```

Comment:
I think given the data model we have, this code is right
But I am still sad about having a `sourceType` field and then splitting the data into different _tables_ based on that field!